buildbot/configure.py

```diff
@@ -40,6 +40,7 @@ def do_configure(args):
     llvm_enable_lld = 'OFF'
 
     sycl_enable_xpti_tracing = 'ON'
+    xpti_enable_werror = 'ON'
```

Comment:
Shouldn't we put it under `if ags.ci_defaults`?

Reply:
I'd like to do so once `--ci-defaults` is passed to all CI jobs.